cypress/support/stubLocation.js

```diff
@@ -0,0 +1,15 @@
+const stubLocation = options => ({
+  onBeforeLoad(win) {
+    const stubLocation = {
+      coordinates: {
```

Comment:
You have to change this key to coords.

Reply:
@tochman  Yes I see now. . .after reading the npm docs @ https://www.npmjs.com/package/react-geolocated
![Screenshot from 2019-08-25 20-43-30](https://user-images.githubusercontent.com/17458458/63654307-32dfaa80-c779-11e9-8edb-c5c8b1a7d0e6.png)

Reply:
@erikwesterberg ^^^

Reply:
Yes, that's one way of finding that out. Other is to use a debugger and check what the response is.